docs/CHANGELOG.rst

```diff
@@ -16,10 +16,12 @@ Changed
   ``trimmomatic-paired`` processes
 - Bump STAR aligner version in ``resolwebio/rnaseq`` docker image to 2.5.4b
 - Bump Primerclip version in ``resolwebio/dnaseq`` docker image
+- **BACKWARD INCOMPATIBLE:** Refactor ``BBDuk`` process
```

Comment:
Tadej said that `**BACKWARD INCOMPATIBLE:**` needs to be on the top of changelog section.
https://github.com/genialis/resolwe-bio/pull/522#discussion_r177393288